packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -339,3 +339,74 @@ export function attemptToDispatchEvent(
   // We're not blocked on anything.
   return null;
 }
+
+function getEventPriorityForPluginSystem(
+  domEventName: DOMEventName,
+): EventPriority {
+  switch (domEventName) {
```

Comment:
Copypasta from the list except that it doesn't need Default ones. Because they're ... default.